index.html

```diff
@@ -1 +1,147 @@
+                <h2 class="welcome">"Welcome"</h2>
+                <div class="main-header">
+                    <h1 class="h1_1">Learn It...</h1>
+                    <h1 class="h1_2">Solve It...</h1>
+                </div>
+                <p class="description">An educative learning app for both students and teachers. This app Contains slides and questions on general knowledge as well as engineering</p>
+                <p class="quote">"Failure is an option here. If things are not failing, you are not innovating enough."</p>
+                <p class="elon">(Elon Musk)</p>
+            </div>
+        </div>
+    </section>
+    <section id="courses">
+        <div class="courses-main-container">
+            <div class="courses-container">
+                <h2 class="courses-header">Courses</h2>
+                <div class="course-row">
+                    <div id="course">
+                        <div class="col-1"><img class="course-img" src="/assets/calculating.png" alt="math"></div>
+                        <div class="col-2"><p class="c2-text">Mathematics</p></div>
+                        <div class="col-3"><p class="c3-text">Learn basic and engineering mathematics-solve trivial questions in addition</p></div>
+                    </div>
+                    <div id="course">
+                        <div class="col-1"><img class="course-img" src="/assets/eng.png" alt="math"></div>
+                        <div class="col-2"><p class="c2-text">English</p></div>
+                        <div class="col-3"><p class="c3-text">Learn basic and engineering mathematics-solve trivial questions in addition</p></div>
+                    </div>
+                    <div id="course">
+                        <div class="col-1"><img class="course-img" src="/assets/piston.png" alt="math"></div>
+                        <div class="col-2"><p class="c2-text">Mechanical Engineering</p></div>
+                        <div class="col-3"><p class="c3-text">Learn basic and engineering mathematics-solve trivial questions in addition</p></div>
+                    </div>
+                    <div id="course">
+                        <div class="col-1"><img class="course-img" src="/assets/electrical.png" alt="math"></div>
+                        <div class="col-2"><p class="c2-text">Electrical Engineering</p></div>
+                        <div class="col-3"><p class="c3-text">Learn basic and engineering mathematics-solve trivial questions in addition</p></div>
+                    </div>
+                </div>
+                <button type="button" class="course-btn">Join the Community Now</button>
+                <a href="#link" class="whole-program">See the whole program</a>
+            </div>
+        </div>
+    </section>
+    <section id="features">
+        <h2 class="features-header">Featured Lecturers</h2>
+        <div id="feature-container">
+            <div id="feature-col1">
 
+            </div>
```

Comment:
[REQUIRED] Your speakers' section is now overlapping the courses section and it is. also overlapping the partners' section. Kindly fix this so that your website can match the template

![image](https://user-images.githubusercontent.com/62938735/226114206-10983160-95a1-4bba-a9e7-1e03c4e86519.png)

![image](https://user-images.githubusercontent.com/62938735/226114252-6edc8759-7229-4145-9acb-ab2e514dc010.png)